configs/droid.kmap

```diff
@@ -22,3 +22,4 @@ keycode 226 = ToggleCallHangup
 keycode 256 = ToggleCallHangup
 keycode 528 = CameraFocus
 keycode 766 = Camera
+keycode 457 = Camera
```

Comment:
Should the keycode number match the one in https://github.com/sailfishos/xkeyboard-config/pull/3 ? (Probably not, because you said that values shouldn't matter too much:) however 457 and 465 share 2 out of 3 digits:))

Reply:
Not as exact value. The xkeyboard-config evdev mapping has this offset of 8 on everything.